bentoml/handlers/dataframe_handler.py

```diff
@@ -72,15 +73,15 @@ def __init__(
             )
 
     def _get_type(self, item):
-        if item.startswith('int'):
-            return 'integer'
-        if item.startswith('float') or item.startswith('double'):
-            return 'number'
-        if item.startswith('str') or item.startswith('date'):
-            return 'string'
-        if item.startswith('bool'):
-            return 'boolean'
-        return 'object'
+        if item.startswith("int"):
```

Comment:
is this formatted by black? I thought it should be single quote?

Reply:
Yes, those formatted by black.  I will re-run it to make sure.

Reply:
changed back.  format script skip string normalization